public/modules/custom/grants_profile/src/UserRedirect.php

```diff
@@ -36,8 +36,10 @@ class UserRedirect implements UserRedirectInterface {
    * @param \Drupal\Core\Session\AccountProxyInterface $current_user
    *   The current active user.
    */
-  public function __construct(RequestStack $request_stack,
-                              AccountProxyInterface $current_user) {
+  public function __construct(
+    RequestStack $request_stack,
+    AccountProxyInterface $current_user,
```

Comment:
These should use constructor promotion (https://stitcher.io/blog/constructor-promotion-in-php-8)